content/patterns/button/button-pattern.html

```diff
@@ -14,93 +14,94 @@
-            It is important that both the appearance and role of a widget match the function it provides.
-            Nevertheless, elements occasionally have the visual style of a link but perform the action of a button.
-            In such cases, giving the element role <code>button</code> helps assistive technology users understand the function of the element.
-            However, a better solution is to adjust the visual design so it matches the function and ARIA role.
+            ボタンが実行するアクションの種類は、リンクの機能（<a href="../link/link-pattern.html">リンクパターン</a>を参照）とは明確に異なります。
+            ウィジェットの外観とロールの両方が、提供する機能と一致することが重要です。
+            それにもかかわらず、要素は時折、リンクの視覚スタイルを持ちながら、ボタンのアクションを実行することがあります。
+            そのような場合、要素に <code>button</code> ロールを与えることは、支援技術利用者が要素の機能を理解するのに役立ちます。
+            ただし、より良い解決策は、機能と ARIA ロールに一致するように視覚デザインを調整することです。
           </p>
         </div>
       </section>
 
       <section id="examples">
         <img alt="" src="../../images/pattern-button.svg">
-        <h2>Examples</h2>
+        <h2>例</h2>
         <ul>
-          <li><a href="examples/button.html">Button Examples</a>: Examples of clickable HTML <code>div</code> and <code>span</code> elements made into accessible command and toggle buttons.</li>
+          <li><a href="examples/button.html">ボタンの例</a>: アクセシブルなコマンドボタン及びトグルボタンにされたクリック可能な HTML <code>div</code> 及び <code>span</code> 要素の例。</li>
           <li>
-            <a href="examples/button_idl.html">Button Examples (IDL)</a>: Examples of clickable HTML <code>div</code> and <code>span</code> elements made into accessible command and toggle buttons.
-            This example uses the <a class="specref" href="#idl-interface">IDL Interface</a>.
+            <a href="examples/button_idl.html">ボタンの例（IDL バージョン）</a>: アクセシブルなコマンドボタン及びトグルボタンにされたクリック可能な HTML <code>div</code> 及び <code>span</code> 要素の例。
+            この例では、<a class="specref" href="#idl-interface">IDL インターフェース</a>を使用しています。
           </li>
         </ul>
       </section>
 
       <section id="keyboard_interaction">
-        <h2>Keyboard Interaction</h2>
-        <p>When the button has focus:</p>
+        <h2>キーボード操作</h2>
+        <p>ボタンにフォーカスがある場合:</p>
         <ul>
-          <li><kbd>Space</kbd>: Activates the button.</li>
-          <li><kbd>Enter</kbd>: Activates the button.</li>
+          <li><kbd>Space</kbd>: ボタンを作動する。</li>
+          <li><kbd>Enter</kbd>: ボタンを作動する。</li>
           <li>
-            Following button activation, focus is set depending on the type of action the button performs. For example:
+            ボタンの作動後、フォーカスはボタンが実行するアクションの種類に応じて設定されます。例:
             <ul>
               <li>
-                If activating the button opens a dialog, the focus moves inside the dialog.
-                (see <a href="../dialog-modal/dialog-modal-pattern.html">dialog pattern</a>)
+                ボタンをアクティブにするとダイアログが開く場合、フォーカスはダイアログ内に移動します。
```

Comment:
「ボタンをアクティブにする」は不自然な表現だし、すぐ上で「ボタンを作動する」と訳しているので、表現を揃えたほうがよさそう。
```suggestion
                ボタンを作動するとダイアログが開く場合、フォーカスはダイアログ内に移動します。
```

Reply:
~~「作動」に揃えます！わたしの方で揃えてコミットします🙏~~

Reply:
すみません、改めて[用語集](https://docs.google.com/spreadsheets/d/1V8wX-pxAO-zuYwTSvTSuZ_FtnV47su6Tyy2vM5GEOLw/edit?gid=0#gid=0&range=51:51)を見ると、activateの対訳は「動作」とするようです
「作動」という議論もあるとありますが、いったん用語集に揃えてしまいます

「ボタンを動作させる」の形になりますかね。

> 英語 | 対応する日本語 | 日本語のNG例 | 備考
> -- | -- | -- | --
> Activate | 動作 | 操作<br>押下<br>アクティベート | 「押下」という言葉は使用しない ( push の意味であっても使用しないこと)「作動」とするのが良いのではないかとする議論もある